### DISCUSSION TITLE
New Character from Als Character Example

### DISCUSSION THREAD

**Discussion by HerrscherOfLolly** [Sixze/ALS-Refactored#236] [Q&A]
Hi, I ran into a new problem when creating my character by inheriting from a ready-made instance. After making the basic settings for the controls and connecting the mesh, I just can't get the camera to move. The character itself moves and does all other actions, but the camera is just static. What could be the problem? I observed a similar defect, if in the used character of your plugin just change the mesh.

**Comment by Sixze** [comment]
The camera component inherits from the skeletal mesh component, so for it to work, you need to assign a camera mesh and an animation blueprint in the component's settings.
